packages/wallets/src/components/Base/WalletButton/WalletButton.tsx

```diff
@@ -10,7 +10,7 @@ interface WalletButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
     isFullWidth?: boolean;
     isRounded?: boolean;
     size?: Extract<TGenericSizes, 'lg' | 'md' | 'sm'>;
-    text?: React.ReactNode;
+    text?: string;
```

Comment:
Note: `text` must be `string` instead of `ReactNode`. @aizad-deriv please confirm